builtin/logical/database/dbplugin/plugin.go

```diff
@@ -32,9 +33,9 @@ type Database interface {
 
 // PluginFactory is used to build plugin database types. It wraps the database
 // object in a logging and metrics middleware.
-func PluginFactory(ctx context.Context, pluginName string, sys pluginutil.LookRunnerUtil, logger log.Logger) (Database, error) {
+func PluginFactory(ctx context.Context, pluginName string, pluginType consts.PluginType, sys pluginutil.LookRunnerUtil, logger log.Logger) (Database, error) {
```

Comment:
I don't think we need the `pluginType consts.PluginType` param in here. We can simply pass it in, `sys.LookupPlugin(ctx, pluginNameconsts.PluginTypeDatabase)`, since the expected lookup for this func is always a database plugin.

Reply:
Oh yes, excellent point. Will do.

Reply:
This has been pushed.